conf/discovery.yaml.template

```diff
@@ -0,0 +1,3 @@
+DISCOVERY:
```

Comment:
Looks good. I would however add a description of when the discovery iso is required and for what kinds of tests, For documentation purposes.

Reply:
I'm not going to block merge on this, as I think there are other places for this to be documented or found.